fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/impl/client/rendering/fluid/DefaultLikeFluidRenderer.java

```diff
@@ -0,0 +1,382 @@
+package net.fabricmc.fabric.impl.client.rendering.fluid;
+
+import net.fabricmc.fabric.api.client.render.fluid.v1.CustomFluidRenderer;
+import net.fabricmc.fabric.api.client.render.fluid.v1.FluidOverlayBlock;
+import net.fabricmc.fabric.api.client.render.fluid.v1.FluidRenderHandler;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.LeavesBlock;
+import net.minecraft.block.TransparentBlock;
+import net.minecraft.client.render.VertexConsumer;
+import net.minecraft.client.render.WorldRenderer;
+import net.minecraft.client.render.model.ModelLoader;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.tag.FluidTags;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.util.shape.VoxelShape;
+import net.minecraft.util.shape.VoxelShapes;
+import net.minecraft.world.BlockRenderView;
+import net.minecraft.world.BlockView;
+
+public class DefaultLikeFluidRenderer implements CustomFluidRenderer {
+	private static final float HALF_PI = MathHelper.field_29845;
+
+	@Override
+	public boolean renderFluid(BlockPos pos, BlockRenderView world, VertexConsumer vertexConsumer, FluidState state, DefaultBehavior defaultBehavior) {
+		return render(world, pos, vertexConsumer, state);
+	}
+
+	private static final float DEFAULT_FLUID_HEIGHT = 8 / 9f;
```

Comment:
Could be changed by overridable method

Reply:
If the render method is native to this class and not referenced by and external classes there is no point specifying it separately if renderfluid is a passthrough to it.

Might as well move the body of render to renderFluid to minimize the code a bit.